.kokoro/presubmit/samples.cfg

```diff
@@ -14,13 +14,14 @@ env_vars: {
 # TODO: remove this after we've migrated all tests and scripts
 env_vars: {
   key: "GCLOUD_PROJECT"
-  value: "gcloud-devel"
+  value: "java-docs-samples-testing"
 }
 
 env_vars: {
   key: "GOOGLE_CLOUD_PROJECT"
-  value: "gcloud-devel"
+  value: "java-docs-samples-testing"
 }
+
 env_vars: {
     key: "GOOGLE_APPLICATION_CREDENTIALS"
     value: "keystore/73713_java_it_service_account"
```

Comment:
I think you have to update GOOGLE_APPLICATION_CREDENTIALS too . [with java-docs-samples creds]